database/forms.py

```diff
@@ -56,6 +57,23 @@ class UserForm(CrispyModelForm):
         last_name = CharField(max_length=256)
         email = EmailField()
 
+        def clean(self):
+            cleaned_data = super().clean()
+            last_name = cleaned_data.get("last_name")
+            first_name = cleaned_data.get("first_name")
+            email = cleaned_data.get("email")
+            if any(i.isdigit() for i in first_name):
+                raise ValidationError("First Name must contain only characters.")
+            if any(i.isdigit() for i in last_name):
+                raise ValidationError("Last Name must contain only characters.")
+            if BaseUser.objects.filter(username=email).exists():
```

Comment:
these lines seem redundant / I'm not sure what they do. Wouldn't you only need the first check? It seems to me that BaseUser.objects.filter(email=email).exists() should be sufficient

Reply:
there is an error if for example - example @ user.com is registered and I try to register example @gmail.com

- reason for error - email is split before the @ and the name before the @ has to be unique 
- I believe the split before the @ creates the username.
- Im sure there is a little bit of redundancy but it begs the question if we should change how the username is created or

Reply:
I hadn't considered that, but also I wonder if this is a circumstance we will run into in a live setting? I'd imagine all the emails the db manages will be @drew.edu

Reply:
yeah thats fair. do you think it hurts to have a little bit of redundancy in here? Maybe if the admin fat fingers an email or makes a human error, that this could catch it?